easybuild/tools/filetools.py

```diff
@@ -655,7 +655,7 @@ def guess_patch_level(patched_files, parent_dir):
     return patch_level
 
 
-def apply_patch(patch_file, dest, fn=None, copy=False, level=None):
+def apply_patch(patch_file, dest, fn=None, copy=False, level=None, log_ok=True, from_pr=None, obliged=False):
```

Comment:
I don't think `log_ok` is very useful, we should always run the `patch` command with `log_ok=False` since we check the return value of `run_cmd` anyway?